documentation/src/main/wiki/Assembler.md

```diff
@@ -130,8 +133,14 @@ Detailed description of each configuration option is described in the next subse
 
 ### Fragment Assembler config
 
-| Name                        | Type                                | Mandatory      | Description  |
-|-------:                     |:-------:                            |:-------:       |-------|
-| `address`                   | `String`                            | &#10004;       | Event bus address of the Fragment Assembler verticle. |
-| `unprocessedStrategy`       | `String`                             | &#10004;       | Strategy for unprocessed Fragments (`AS_IS`, `UNWRAP`, `IGNORE`). `UNWRAP` is default strategy if no strategy defined. |
+| Name                        | Type      | Mandatory      | Description  |
+|-------:                     |:-------:  |:-------:       |-------|
+| `address`                   | `String`  | &#10004;       | Event bus address of the Fragment Assembler verticle. |
+| `unprocessedStrategy`       | `String`  | &#10004;       | Strategy for unprocessed Fragments (`AS_IS`, `UNWRAP`, `IGNORE`). `UNWRAP` is default strategy if no strategy defined. |
+| `snippetTagName`            | `String`  | &#10004;       | The name of a tag that will be recognised as a Knot.x snippet. Remember to update [[Splitter configuration\|Splitter#how-to-configure]] |
 
+**Important - whenever you change `snippetTagName` to custom one remember that Knot.x splits 
```

Comment:
I'd rephrase a little bit the notice, just to warn user about what not to do in the first sentence:
```
Important - When specifying `snippetTagName` remember to not use standard HTML tags like `div`, `span`, etc.
Knot.x splits an HTML into fragments by parsing it as String to get best possible performance. 
It simply search text for the opening and first matching closing tag. It does not analyse the text as HTML. So, if you use `div` as fragmentTagName, and inside your will use multiple `div` tags too, then it will not pick the one that matches first opening, instead it will get the fragment up to the first closing `div` tag. It will result in a broken HTML structure.
```